proto/server.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
```

Comment:
Here is a solid setup:

https://github.com/stevvooe/continuity/blob/master/proto/gen.go

From the repo root, I can run `go generate ./proto` and it will "do the needful". It let's you get all the nasty details that might come with generation out of the Makefile.